saleor/payment/gateways/np_atobarai/api_helpers.py

```diff
@@ -0,0 +1,250 @@
+        )
+        if 400 < response.status_code <= 600:
+            raise requests.HTTPError
+        return response
+
+
+def np_request(
+    config: "ApiConfig", method: str, path: str = "", json: Optional[dict] = None
+) -> NPResponse:
+    try:
+        response = _request(config, method, path, json)
+        response_data = response.json()
+        if "errors" in response_data:
+            return NPResponse({}, response_data["errors"][0]["codes"])
+        return NPResponse(response_data["results"][0], [])
+    except requests.RequestException:
+        msg = "Cannot connect to NP Atobarai."
+        logger.warning(msg, exc_info=True)
+        return NPResponse({}, [msg])
+
+
+def handle_unrecoverable_state(
+    action: str,
+    transaction_id: str,
+    error_codes: Iterable[str],
+) -> None:
+    logger.error(
+        "Payment #%s %s Unrecoverable Error: %s",
+        transaction_id,
+        action,
+        ", ".join(error_codes),
+    )
+
+
+def health_check(config: "ApiConfig") -> bool:
+    try:
+        _request(config, "post", "/authorizations/find")
+        return True
+    except requests.RequestException:
+        return False
+
+
+def format_name(ad: AddressData) -> str:
+    """Follow the Japanese name guidelines."""
+    return f"{ad.first_name} {ad.last_name}".strip()
+
+
+def format_address(config: "ApiConfig", ad: AddressData) -> Optional[str]:
+    """Follow the Japanese address guidelines."""
+    # example: "東京都千代田区麹町４－２－６　住友不動産麹町ファーストビル５階"
+    if not config.fill_missing_address:
+        return f"{ad.country_area}{ad.street_address_2}{ad.street_address_1}"
+    with Posuto() as pp:
+        try:
+            jap_ad = pp.get(ad.postal_code)
+        except KeyError:
+            return None
+        else:
+            return (
+                f"{ad.country_area}"
+                f"{jap_ad.city}"
+                f"{jap_ad.neighborhood}"
+                f"{ad.street_address_2}"
+                f"{ad.street_address_1}"
+            )
+
+
+def format_price(price: Decimal, currency: str) -> int:
+    return int(price_to_minor_unit(price, currency))
+
+
+def get_refunded_goods(
+    refund_data: Dict[int, int],
+    payment_information: PaymentData,
+) -> List[dict]:
+    return [
+        {
+            "goods_name": payment_line.product_name,
+            "goods_price": format_price(
+                payment_line.gross, payment_information.currency
+            ),
+            "quantity": quantity,
+        }
+        for payment_line in payment_information.lines
+        if (quantity := refund_data.get(payment_line.variant_id, payment_line.quantity))
+    ]
+
+
+def get_goods(payment_information: PaymentData) -> List[dict]:
+    return [
+        {
+            "quantity": line.quantity,
+            "goods_name": line.product_name,
+            "goods_price": format_price(line.gross, payment_information.currency),
+        }
+        for line in payment_information.lines
+    ]
+
+
+def get_discount(
```

Comment:
Not sure if the name is correct. I see that it is used in some transactions like refund or partial returns. 
And have assigment like:
```
goods = get_discount(payment_information)
```
which suggest that `get_discount` returns a goods data.

Reply:
I assume that `-` in goods_price for the last element is related to places where it is used - trans refund/return. So I believe this should be included somehow in the name of the function